docs/python/jupyter-support.md

```diff
@@ -11,134 +11,136 @@ MetaSocialImage: images/tutorial/social.png
+The selected code cell can be changed using the mouse, the up/down arrow keys on the keyboard, and the `kbstyle(J)` (up) and `kbstyle(K)` (down) keys. When you select a code cell, there are two different modes.....
 
-You can also click in the margin to the left of line numbers to set breakpoints. Then you can use **Debug Cell** to start a debugging sessions for that code cell. The debugger stops execution at breakpoints and allows you to step through code one line at a time and inspect variables (see [Debugging](debugging.md) for details).
+### Run a single code cell
 
-## Python Interactive window
+Once your code is added, you can run the cell using the green run arrow and the output will be displayed below the code cell.
 
-The **Python Interactive** window, mentioned in the previous section, can be used as a standalone console with arbitrary code (with or without code cells).
-To use the window as a console, open it with the **Python: Show Python Interactive window** command from the Command Palette. You can then type in code, using `kbstyle(Enter)` to go to a new line and `kbstyle(Shift+Enter)` to run the code.
+![Run Jupyter code cell](images/jupyter/native-code-cells-03.png)
 
-To use the window with a file, use the **Run Current File in Python Interactive window** command from the Command Palette.
+Alternately, you can use the key combos `kbstyle(Ctrl+Enter)` to run the current cell, `kbstyle(Shift+Enter)` to run the current cell and insert a new cell immediately below (focus moves to new cell), or `kbstyle(Alt+Enter)` to run the current cell and insert a new cell immediately below (focus remains on current cell).
 
-### IntelliSense
+### Run multiple code cells
 
-The Python Interactive window has full IntelliSense – code completions, member lists, quick info for methods, and parameter hints. You can be just as productive typing in the Python Interactive window as you are in the code editor.
+Running multiple code cells can be accomplished in a number of ways. You can use the double arrow in the toolbar of the Notebook Editor to run all cells within the Notebook or the hover toolbar arrows to run all cells above or below the current code cell.
 
-![IntelliSense in the Python Interactive window](images/jupyter/interactive-window-intellisense.gif)
+![Run multiple code cells](images/jupyter/native-code-cells-04.png)
 
-### Plot Viewer
+### Move a code cell
 
-The Plot Viewer gives you the ability to work more deeply with your plots. In the viewer you can pan, zoom, and navigate plots in the current session. You can also export plots to PDF, SVG, and PNG formats.
+Moving code cells up or down within a Notebook can be accomplished using the vertical arrows beside each code cell. Simply select the cell by clicking the vertical bar to it's left or by clicking within the cell content. Then click the up arrow to move the cell up and the down arrow to move the cell down.
```

Comment:
You don't actually need to click into the cell first to click the move up and down arrows. They appear on hover of the cell